### PR TITLE
Aea 845 update examples

### PR DESCRIPTION
**Pull request by conner-jowitt**

### CODE REVIEW

**Review by KevinMayfield — COMMENTED** [review]
Checked the MessageHeader payload changes match the specification

File: models/schemas/MessageDestination.yaml [hunk 1]
```diff
@@ -0,0 +1,18 @@
+type: object
+description: "Message destination application"
```
Comment (harryhmc) [inline models/schemas/MessageDestination.yaml:2]: i dont think quotes are needed for descriptions, are they?

File: models/schemas/MessageHeader.yaml [hunk 2]
```diff
@@ -20,21 +21,21 @@ properties:
   eventCoding:
     $ref: "Coding-MessageDefinition.yaml"
   sender:
-    $ref: "Reference.yaml"
+    $ref: "SenderReference.yaml"
   source:
     type: object
     description: "Message source application"
```
Comment (harryhmc) [inline models/schemas/MessageHeader.yaml:27]: not your code, but same as previous comment

**Review by harryhmc — APPROVED** [review]

File: models/schemas/MessageDestination.yaml (hunk 1 above)
Comment (conner-jowitt) [reply models/schemas/MessageDestination.yaml:2]: Not required, but I was matching the style of most of the other schemas. Probably worth raising a ticket to standardise them?

**Comment by conner-jowitt** [comment]
Merged in branch AEA-843 to pass the fhir-validator. I think it requires the changes from this branch

**Review by booshibooshi — APPROVED** [review]

**Pull request merged**